bin/journey-test.sh

```diff
@@ -164,7 +164,7 @@ make_local_trackler() {
   cp -r ${track_root}/exercises tracks/${TRACK}
 
   gem install bundler
-  bundle install
+  bundle install --jobs $((`nproc --all`-1))
```

Comment:
Whatever we do here should also be extracted into a variable so that it can be shared between the two `bundle install` invocations.